dataflow/conftest.py

```diff
@@ -55,6 +55,17 @@ def hyphen_name(name: str) -> str:
     def underscore_name(name: str) -> str:
         return UNDERSCORE_NAME_RE.sub("_", Utils.hyphen_name(name))
 
+    @staticmethod
+    def wait_until(
+        is_done: Callable[[], bool],
+        timeout_sec: int = TIMEOUT_SEC,
+        poll_interval_sec: int = POLL_INTERVAL_SEC,
+    ) -> bool:
+        for _ in range(0, timeout_sec, poll_interval_sec):
+            if is_done():
+                return True
+        return False
```

Comment:
Should this function sleep for `poll_interval_sec` in the loop body?  (Or is the interval baked in somewhere else?)

Reply:
True! Forgot that, thanks for keeping an eye on it :)